lib/queueMicrotask.mjs

```diff
@@ -31,7 +31,8 @@ const createQueueMicrotaskViaMutationObserver = () => {
   let microtaskQueue = [];
   const observer = new MutationObserver(() => {
     microtaskQueue.forEach((microtask) => microtask());
-    microtaskQueue = [];
+    for (let x = 0; x < microtaskQueue.length; ++x) microtaskQueue[x]();
```

Comment:
seems microtaskQueue run twice. You forget  delete forEach tasks.

Reply:
oups, yes, thanks!